### ISSUE TITLE
Authentication Failure

### ISSUE THREAD

**Issue by bhavitvyamalik** [JonathanHolvey/sharepy#46]
I'm getting ```Authentication Failure: AADSTS53003: Access has been blocked by Conditional Access policies. The access policy does not allow token issuance.``` How can I fix it?

**Comment by berlineric** [comment]
same issue here, is it related to any configuration that need to be done on share point side ?

**Comment by RasmusParsmo** [comment]
I have the same issue, perhaps something to do with 2FA?

**Comment by JonathanHolvey** [comment]
After setting up a new SharePoint dev environment yesterday, I started experiencing this issue. Logging in on the web (as the site admin) revealed that:

> Microsoft has enabled Security Defaults to keep your account secure.

You can read about security details in [this article](https://docs.microsoft.com/en-gb/azure/active-directory/fundamentals/concept-fundamentals-security-defaults). Basically, it enforces security rules that appear to be incompatible with SharePy's authentication methods.

The simplest way to resolve this issue would be to disable security defaults, however this might not be an option for everyone.

**Comment by RasmusParsmo** [comment]
That was what I was afraid of, thanks for your help anyway!

**Comment by versex799** [comment]
Just for those that are seeing this error in the future.... in plain English.... I ran into this issue and it was because the account was blocked. I had the correct password but Microsoft flagged the login attempt as suspicious and blocked it.

**Comment by wUEZRs** [comment]
We saw this in the user's sign in logs. We found more info under Activity Details: Sign-ins > Conditional Access

The user was in this case traveling to a geo blocked IP
